test/Farmer/addProduct.test.js

```diff
@@ -0,0 +1,53 @@
+import { expect } from "chai";
+import nock from "nock";
+import { describe, it } from "mocha";
+import { BASEURL } from "../../src/core/urls.js";
+import { getUserFarmsSuccess, userFarmsAreNil } from "./farmerResponses.js";
+import categoriesFound, {
+  categoriesServerError,
+} from "./productcallResponses.js";
+import { fetchCategories } from "../../src/products/productmanagement.js";
+import { getUserFarms } from "../../src/users/farmer/farmmanagement.js";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
import { getUserFarms } from '../../src/users/farmer/farmmanagement.js';
```